tests/lib/tools/snaps-state

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash -e
+
+show_help() {
+    echo "usage: pack-local <snap-name>"
+    echo "       install-local <snap-name> [OPTIONS]"
+    echo "       install-local-as <snap-name> <dest-name> [OPTIONS]"
+    echo "       is-classic-confinement-supported"
```

Comment:
I would honestly leave that out, it's unrelated to the rest and we have `snap debug sandbox-features --required confinement-options:classic` which, while mouthful, does exactly what we need to know from a first-party snap command already.

Reply:
sure, makes sense